p/ext.php

```diff
@@ -48,7 +53,12 @@ function is_valid_path_extension($path, $extensionPath) {
 		return false;
 	}
 
-	// File to serve must be under a `ext_dir/static/` directory.
+	// User files do not need further validations
+	if (!$isStatic) {
```

Comment:
We should not allow arbitrary paths in the filesystem, so `$path` needs to be constrained somehow, at least preventing things such as `../`

Reply:
This validation is skipped only for files generated by the extension configuration in the user folder. And the validation specifies that it must be in the user folder.
I am confused why we should be testing static for those files. Because in the event of the use of `../`, it has to be contained in the user folder. So I guess the risks are really low.